test_scenarios/data/T4_in.puml

```diff
@@ -33,9 +33,57 @@ object "testTask01 : ComputationTask" as testTask01 {
 
 object "input01 : ComputationInput" as input01 {
     logger = null
+    n = 20
 }
 
 testApp01 --o testTask01
 testTask01 o-- input01
 
+object "testApp02 : ComputationApplication" as testApp02 {
+    id = <any>
+    name = "Test App 02"
+    description = "Fibonnacci sum"
+    icon = "https://static.thenounproject.com/png/1751844-200.png"
+}
+
+object ": ComputationStepPackage" as pack2 {
+    applicationId = <any>
+    computationSteps = [step21, step22]
+    version: v1.0.0
+}
+
+testApp02 <.. pack2
+
+object "step21 : ComputationStep" as step21 {
+    params = []
+    artifactUrl = <<app counting phi sequence from 1 to n and saving it to file>>
+    - command
+}
+
+object "step22 : ComputationStep" as step22 {
+    params = []
+    artifactUrl = <<app counting sum of the series saved in a file>>
+    - command
+}
+
+pack2 *-- step21
+pack2 *-- step22
+
+object "testTask02 : ComputationTask" as testTask02 {
+    id = <any>
+    name = "Test Task 02"
+    userId = 00001
```

Comment:
Możesz usunąć definicję userId? Tak trochę nie mamy na to wpływu, może po prostu napisz "jak użytkownika testowego"? <as_test_user_id> na przykład

Reply:
Poprawione. OK?